src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -181,7 +181,7 @@ where
             W: vec![C::default(); num_witness],
             E: C::default(),
             u: C::Scalar::ZERO,
-            challenges: iter::repeat_with(|| BigUint::one(limb_width))
+            challenges: iter::repeat_with(|| BigUint::zero(limb_width))
```

Comment:
Yes, this is a bug I found during test. So I changed the chip default value as well. For some custom gates, if the default value of challenge is one, the default E might not be zero anymore which will break the folded relation. If everything is zero, it would be fine.